test/connection_test.rb

```diff
@@ -411,6 +411,14 @@ def test_init_with_block_yields_connection
     assert_equal '/omnom', conn.path_prefix
   end
 
+  def test_respond_to
+    assert Faraday.default_connection.respond_to?(:get)
```

Comment:
These 2 assertions are not necessary. You didn't make any changes to `default_connection`, why would you add assertions that verify its interface?